lib/internal/vm/localworker.js

```diff
@@ -0,0 +1,139 @@
+'use strict';
+
+// LocalWorker was originally a separate module developed by
+// Anna Henningsen and published separately on npm as the
+// synchronous-worker module under the MIT license. It has been
+// incorporated into Node.js with Anna's permission.
+// See the LICENSE file for LICENSE and copyright attribution.
+
+const {
+  Promise,
+} = primordials;
+
+const {
+  LocalWorker: LocalWorkerImpl,
+} = internalBinding('contextify');
+
+const EventEmitter = require('events');
+const { setTimeout } = require('timers');
+const { dirname, join } = require('path');
+
+let debug = require('internal/util/debuglog').debuglog('localworker', (fn) => {
+  debug = fn;
+});
+
+class LocalWorker extends EventEmitter {
+  #handle = undefined;
+  #process = undefined;
+  #global = undefined;
+  #module = undefined;
+  #stoppedPromise = undefined;
+
+  /**
+   */
+  constructor() {
+    super();
+    this.#handle = new LocalWorkerImpl();
+    this.#handle.onexit = (code) => {
+      this.stop();
+      this.emit('exit', code);
+    };
+    try {
+      this.#handle.start();
+      this.#handle.load((process, nativeRequire, globalThis) => {
+        this.#process = process;
+        this.#module = nativeRequire('module');
+        this.#global = globalThis;
+        process.on('uncaughtException', (err) => {
+          if (process.listenerCount('uncaughtException') === 1) {
+            // If we are stopping, silence all errors
+            if (!this.#stoppedPromise) {
+              this.emit('error', err);
```

Comment:
```suggestion
              this.emit('uncaughtException', err);
```
nitpick. makes more sense IMO